.github/workflows/ci.yml

```diff
@@ -40,10 +40,10 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
-      - name: "KomaMRICore dev setup"
+      - name: "KomaMRICore generate jld2 test files"
         shell: bash
         run: |
-          julia --color=yes --project="KomaMRICore" -e "using Pkg; Pkg.develop(PackageSpec(; path=\"./KomaMRIFiles\"));"
+          julia --color=yes --project="KomaMRIFiles" -e "using Pkg; Pkg.develop(PackageSpec(; path=\"./KomaMRICore\")); include(\"./KomaMRIFiles/test/generate_core_test_files/run.jl\")"
```

Comment:
Just saw this. This doesn't make sense. If someone did `test` in their pc the package would not pass the tests.

Reply:
Addessed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/bde381a2d17204903291246e0636c4bbdf961651

Reply:
I do not see it addressed in that commit

Reply:
Sorry, addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/e37397a7c4d95cc942a71195de6c52d1aeacfe01